extensions/GuidedBugEntry/template/en/default/guided/guided.html.tmpl

```diff
@@ -229,48 +232,59 @@ tabbed browsing or the location bar)
 [%############################################################################%]
 
 [% BLOCK exits %]
-<table class="exits">
-  <tr>
-    <td>
-      <div class="exit_img">
+  <div class="options">
+    <div class="option">
+      <div class="icon">
         <a href="https://support.mozilla.org/">
-        <img src="[% basepath FILTER none %]extensions/GuidedBugEntry/web/images/support.png" width="32" height="32"></a>
+          <!-- Material Symbols: Contact Support -->
+          <svg xmlns="http://www.w3.org/2000/svg" height="48" width="48"><path d="m25.75 44-.5-5.5h-2q-7.15 0-12.2-5.05Q6 28.4 6 21.25q0-7.15 5.075-12.2Q16.15 4 23.35 4q3.55 0 6.575 1.275Q32.95 6.55 35.2 8.85q2.25 2.3 3.525 5.425T40 21.1q0 3.3-.975 6.6T36.2 34q-1.85 3-4.5 5.55T25.75 44Zm-2.4-11.15q.8 0 1.35-.55t.55-1.35q0-.8-.55-1.35t-1.35-.55q-.8 0-1.35.55t-.55 1.35q0 .8.55 1.35t1.35.55ZM22 26.05h2.5q0-1.25.425-2.075.425-.825 1.775-2.175 1.35-1.35 1.9-2.475.55-1.125.55-2.425 0-2.25-1.525-3.7-1.525-1.45-4.075-1.45-2.1 0-3.75 1.1t-2.45 3l2.3.95q.55-1.3 1.525-1.925.975-.625 2.225-.625 1.5 0 2.35.725.85.725.85 1.925 0 .95-.55 1.925-.55.975-1.95 2.425-1.35 1.4-1.725 2.25-.375.85-.375 2.55Z"/></svg>
```

Comment:
Are these SVGs copied from another location? The bug/commit message mention changing the color of the button, is this relevant?

Can you post a screenshot showing the change?

Reply:
The icon on the home page changes based on the light/dark setting of the OS which was not happeninging on the Guided entry product page. This change makes both pages act the same way.

Before:
![image](https://github.com/user-attachments/assets/561ca679-9077-484c-acbb-d57719851396)

After:
![image](https://github.com/user-attachments/assets/a53d7e9e-eb0b-462b-bd3a-461f4ce6645f)